src/sentry/models/groupsubscription.py

```diff
@@ -34,22 +34,33 @@ class GroupSubscriptionManager(BaseManager):
     def subscribe(
         self,
         group: Group,
-        user: User | RpcUser,
+        subscriber: User | RpcUser | Team,
         reason: int = GroupSubscriptionReason.unknown,
     ) -> bool:
         """
-        Subscribe a user to an issue, but only if the user has not explicitly
+        Subscribe a user or team to an issue, but only if that user or team has not explicitly
         unsubscribed.
         """
+        from sentry.models import Team, User
+
         try:
             with transaction.atomic(router.db_for_write(GroupSubscription)):
-                self.create(
-                    user_id=user.id,
-                    group=group,
-                    project=group.project,
-                    is_active=True,
-                    reason=reason,
-                )
+                if isinstance(subscriber, User) or isinstance(subscriber, RpcUser):
+                    self.create(
+                        user_id=subscriber.id,
+                        group=group,
+                        project=group.project,
+                        is_active=True,
+                        reason=reason,
+                    )
+                elif isinstance(subscriber, Team):
```

Comment:
@isabellaenriquez what about `RpcTeam`?

Reply:
good catch!

Reply:
ok update @scefali, i just realized that there doesn't seem to be a way to unpack team members from an RpcTeam instance, which is an issue when the feature flag isn't enabled (RpcTeam was never included in the first place, so this wasn't dealt with before).

Reply:
There doesn't appear to be any instances where `subscribe_actor` would be called with an RpcTeam either.